### PR TITLE
Fix homework review jumpiness -- make it animate nicely

### PR DESCRIPTION
**Pull request by pandafulmanda**
### Step should now move to completed list as completed

out

### CODE REVIEW

**Comment by pandafulmanda** [comment]
upon further review, this is not a bug, and may actually be a change request

**Comment by pandafulmanda** [comment]
Although, this should be merged for the fixes it provides, and the added attention to the experience, and we can make a separate PR for the specific behavior of the button on the review page

File: src/components/task-step/ends.cjsx [hunk 1]
```diff
@@ -86,6 +96,7 @@ HomeworkEnd = React.createClass
           goToStep={@goToStep}
           onNextStep={@onNextStep}
           review={type}
+          ref={"task-review-#{type}"}
```
Comment (philschatz) [inline src/components/task-step/ends.cjsx]: Is this ref used anywhere?

Comment (pandafulmanda) [inline src/components/task-step/ends.cjsx]: oh...hmm...i may have been trying to make that a key actually because of warnings about keys...

**Pull request merged**